Packs/Core/Playbooks/playbook-Remote_WMI_Process_Execution.yml

```diff
@@ -0,0 +1,1004 @@
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 2420
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+      replyOptions:
+      - Isolate
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
+    id: "18"
+    taskid: 7df12c62-a960-428c-8e0f-dccf404b63e0
+    type: regular
+    task:
+      id: 7df12c62-a960-428c-8e0f-dccf404b63e0
+      version: -1
+      name: Isolate endpoint
+      description: Isolates the specified endpoint.
+      script: '|||core-isolate-endpoint'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "14"
+    scriptarguments:
+      endpoint_id:
+        simple: ${Endpoint.ID}
+      incident_id:
+        complex:
+          root: alert
+          accessor: parentXDRIncident
+          transformers:
+          - operator: Cut
+            args:
+              delimiter:
+                value:
+                  simple: '-'
+              fields:
+                value:
+                  simple: "2"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 2605
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: f4474c65-78f5-4acd-8954-1ed6559bc89e
+    type: condition
+    task:
+      id: f4474c65-78f5-4acd-8954-1ed6559bc89e
+      version: -1
+      name: Should proceed to isolate the remote endpoint?
+      description: If the condition "Check for high-confidence evidence or malicious IP address" was matched and the remote endpoint ID is available, an endpoint isolation is suggested.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "14"
+      "Yes":
+      - "17"
+    separatecontext: false
+    conditions:
+    - label: "Yes"
+      condition:
+      - - operator: AnyMatch
+          left:
+            value:
+              simple: CommandLineAnalysis.findings
+            iscontext: true
+          right:
+            value:
+              simple: mixed case powershell, reversed command, powershell suspicious patterns, credential dumping, double encoding, amsi techniques, malicious commands
+        - operator: greaterThanOrEqual
+          left:
+            value:
+              simple: DBotScore.Score
+            iscontext: true
+          right:
+            value:
+              simple: "2"
```

Comment:
Also here, according to the Confluence page, the IP DBot score in this condition should be 3 (malicious) rather than 2 (suspicious).